llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -117,14 +117,21 @@ struct PortableMemInfoBlock {
   void clear() { *this = PortableMemInfoBlock(); }
 
   // Returns the full schema currently in use.
-  static MemProfSchema getSchema() {
+  static MemProfSchema getFullSchema() {
```

Comment:
Any reason that these are part of the PortableMemInfoBlock? IMO they should be peers of the Schema itself (under llvm::memprof).

Reply:
I'll post a separate patch for that.